onboarding/source/amb_light_service.c

```diff
@@ -4,9 +4,23 @@
 #include <adc.h>
 #include <sci.h>
 
+#include <string.h>
+
+
 /* USER CODE BEGIN */
 // Include any additional headers and global variables here
+#include <FreeRTOS.h>
+#include <os_task.h>
+#include <os_queue.h>
+
+static TaskHandle_t lightServiceTaskHandle = NULL;
+static QueueHandle_t eventQueue = NULL; // CHANGE: more descriptive name for variable
+#define LIGHT_SENSOR  6U // CHANGE: capital
 
+// Helper functions for ADC conversion
+uint16_t getAmbientLightData(void);
+void adcGetSingleData(adcBASE_t *adc, unsigned group, adcData_t *data);
```

Comment:
Typically, a function like this would be useful even outside `amb_light_service.c`, so it'd make sense to declare it in a separate header file that can be included by any source files that'd like to use the function. However, this isn't required for the challenge.

If this was a function that was only going to be used in this file (I.e. it's never called outside `amb_light_service.c`), then it should be declared a static function. Again, not necessary to do this for the challenge.